pkg/resource/image_metadata.go

```diff
@@ -34,15 +34,17 @@ import (
 const (
 	// DefaultExportedPort is the default protocol exposed by inner services specified in image metadata
 	DefaultExportedPort = "http"
-	// LabelKeyPrometheus is the label key for Prometheus metadata
-	LabelKeyPrometheus = "prometheus.io"
 	// LabelKeyOrgKie is the label key for KIE metadata
 	LabelKeyOrgKie = "org.kie" + labelNamespaceSep
 	// LabelKeyOrgKiePersistence is the label key for Persistence metadata
-	LabelKeyOrgKiePersistence = "org.kie" + labelNamespaceSep + "persistence"
+	LabelKeyOrgKiePersistence = LabelKeyOrgKie + "persistence"
+	// LabelKeyOrgKiePersistenceRequired is the label key to check if persistence is enabled or not
+	LabelKeyOrgKiePersistenceRequired = LabelKeyOrgKiePersistence + labelNamespaceSep + "required"
 	// LabelKeyOrgKieProtoBuf is the label key for ProtoBuf metadata
-	LabelKeyOrgKieProtoBuf = "org.kie" + labelNamespaceSep + "persistence" + labelNamespaceSep + "proto"
+	LabelKeyOrgKieProtoBuf = LabelKeyOrgKiePersistence + labelNamespaceSep + "proto"
```

Comment:
same here, . before proto?

Reply:
It's not missing, there's the back slash.